src/test/regress/bin/normalize.sed

```diff
@@ -73,6 +73,9 @@ s/(job_[0-9]+\/task_[0-9]+\/p_[0-9]+\.)[0-9]+/\1xxxx/g
 /^LINE [0-9]+:.*$/d
 /^\s*\^\s*$/d
 
+# Ignore file paths
+s,"/(home|usr|lib)/(.+/)*([^/]*\.so)","\3",
```

Comment:
I don't think this is generic enough. We should hide the error message all-together. You can use something like this function: https://github.com/citusdata/citus/blob/master/src/test/regress/sql/multi_test_helpers.sql#L112